data/lib/src/network/model/converter/data_from_json_converter.dart

```diff
@@ -41,4 +41,8 @@ DocumentId documentIdFromJson(dynamic json) => DocumentId(json.toString());
 
 String mediaTypeToJson(MediaType mediaType) => jsonEncode({Attribute.type: mediaType.mimeType});
 
-MediaType mediaTypeFromJson(dynamic json) => MediaType.parse(json.toString());
\ No newline at end of file
+MediaType mediaTypeFromJson(dynamic json) => MediaType.parse(json.toString());
+
+String sharedSpaceIdToJson(SharedSpaceId sharedSpaceId) => jsonEncode({Attribute.uuid: sharedSpaceId.uuid});
+
+SharedSpaceId sharedSpaceIdFromJson(dynamic json) => SharedSpaceId(json.toString());
```

Comment:
please using `converter`, reference to `MailingListIdDtoConverter` or `ShareIdDtoConverter`